config/env.development

```diff
@@ -87,6 +87,14 @@ JWT_AUDIENCE=http://localhost
 # How long should a JWT work before it expires
 JWT_EXPIRES_IN=1h
 
+# Example Supabase secrets for development
+# NOT INTENDED FOR PRODUCTION
+
+# Supabase connection https://supabase.com/docs/guides/database/connecting-to-postgres
+SUPABASE_API_URL=http://localhost:8911
```

Comment:
In docker, should this not be `http://kong:8911`?  Running in containers locally will not allow `localhost` to work.

Reply:
Right, it shouldn't be localhost, the supabase port is `htttp://kong:8000`. I've tested this locally and it worked.

Reply:
I'm thinking we should publish `htttp://kong:8000` to localhost to, because I don't think `htttp://kong` works on Gitpod.

Reply:
We can do that as a separate thing, but the URL we pass into the container to use has to use the docker network (i.e, container DNS).

It might be good to have an `env.gitpod` separate from `env.development` so this is easier to separate.